### ISSUE TITLE
Multiple PHP warnings when all settings from a Meta List section are disable

### ISSUE THREAD

**Issue by decarvalhoaa** [hyyan/woo-poly-integration#115]
Steps to reproduce:

1. In Admin -> Settings -> WooPoly -> Metas List, disable all settings from one section.
2. Save settings.
3. Open any product and click "Update".

The PHP warnings show up in the error log if enabled.

**Comment by decarvalhoaa** [comment]
The Settings::getOption() in https://github.com/hyyan/woo-poly-integration/blob/woocommerce2.7/src/Hyyan/WPI/Product/Meta.php#L266-L268 returns NULL and array_merge expects arguments to be of type array.

I have checked that the settings-api calss is up to date.

A possible solution is to replace that like with the following:
```
$option = is_array(Settings::getOption($ID, MetasList::getID(), $value['metas'])) ? Settings::getOption($ID, MetasList::getID(), $value['metas']) : $value['metas'];
$metas = array_merge($metas, $option);
```

**Comment by decarvalhoaa** [comment]
Fixed via [8434235](https://github.com/hyyan/woo-poly-integration/commit/8434235486df5d831dba0cdbbb94520592cc42b3).
